packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -738,6 +738,49 @@ describe('plugin-meetings', () => {
     });
 
     describe('#updateSelf', () => {
+      it('should trigger SELF_MEETING_BRB_CHANGED when brb state changed', () => {
+        locusInfo.self = undefined
+
+        const assertBrb = (enabled) => {
+          const selfWithBrbChanged = cloneDeep(self);
+          selfWithBrbChanged.controls.brb = enabled
+
+          locusInfo.emitScoped = sinon.stub();
+          locusInfo.updateSelf(selfWithBrbChanged, [])
+
+          assert.calledWith(
+            locusInfo.emitScoped,
+            { file: 'locus-info', function: 'updateSelf' },
+            LOCUSINFO.EVENTS.SELF_MEETING_BRB_CHANGED,
+            { brb: enabled }
+          )
+        }
+
+        assertBrb(true)
+        assertBrb(false)
+      })
+
+      it('should not trigger SELF_MEETING_BRB_CHANGED when brb state changed', () => {
+        const assertBrb = (enabled) => {
+          const selfWithBrbChanged = cloneDeep(self);
+          selfWithBrbChanged.controls.brb = enabled
```

Comment:
@SomeBody16 I'm not sure if the code style should allow missing parts without a ";". I believe there is something not set up correctly in this logic for JavaScript files, or I’m missing something here.

Reply:
Prettier detects and fixes this automatically for me, though I'm using VS Code. I'd be curious to see why it isn't catching stuff like this for some of y'all.